packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -98,6 +105,14 @@ export const Select = styled( Ariakit.Select, {
 	`
 );
 
+const slideUpAndFade = keyframes( {
+	'0%': {
+		opacity: 0,
+		transform: `translateY(${ ANIMATION_PARAMS.SLIDE_AMOUNT })`,
```

Comment:
Don't we mean to start from minus that amount so we preserve the downward direction?

Reply:
Done, see https://github.com/WordPress/gutenberg/pull/63343#issuecomment-2220162250